src/libstd/fs.rs

```diff
@@ -1724,6 +1734,23 @@ mod tests {
         assert!(canary.exists());
     }
 
+    // FIXME(#12795) depends on lstat to work on windows
+    #[cfg(not(windows))]
```

Comment:
As I outlined in the description of the pull request, I don't know. Just copied it from another test on remove_dir_all. Can you test on windows?